### ISSUE TITLE
Enhancement: Merge settings and changesettings into one

### ISSUE THREAD

**Issue by AstreaTSS** [TheNoob27/starboard-issues#3]
Might as well do more than one suggestion while I'm at it.

Currently, the settings and the changesettings command are seperate. While this is probably easier to code and work with, it has lead to some confusion with users not knowing how to change their settings, as they instinctively think that the settings command should do that.

I suggest that changesettings should be merged with settings, and make its functionality similar to the prefix command, where it displays the setting if no arguments are provided and changes them if there is an argument.

**Comment by TheNoob27** [comment]
Well currently if you don't pass arguments into the changesetting command, the settings command is ran. It actually used to be that the changesetting command would run if arguments were passed into the settings command but then I removed it after I added the ability to view extra info about specific settings.

Maybe I'll add the functionality back but using "set", like `star settings set <settings>`, but that may lead into confusion. ~~The settings are reserved for the extra info though...~~ Actually I could make it if there's a value passed in after the setting then I'd treat it as a changesetting command, like `star setting visible` would view it and `star setting visible false` would change it... I'll think about it
